explorer/templates.go

```diff
@@ -462,6 +462,13 @@ func makeTemplateFuncMap(params *chaincfg.Params) template.FuncMap {
 			dateTime := time.Unix(int64(a), 0).UTC()
 			return dateTime.Format("2006-01-02")
 		},
+		"dateTimeWithoutTimeZone": func(a uint64) string {
```

Comment:
While you're here, would you please remove `timeWithoutDateAndTimeZone`, which seems to be unused and incorrectly named?